include/CppUTest/StandardCLibrary.h

```diff
@@ -1,8 +1,10 @@
 
+/* Must include this first to ensure the StandardC include in CppUTestConfig still happens at the right moment */
+#include "CppUTestConfig.h"
```

Comment:
I second @uecasm's suggestion about inclusion of `CppUTestConfig.h`. 

I am not sure what @basvodde is really trying to say.